### ISSUE TITLE
Remove `babel-runtime` from dependencies

### ISSUE THREAD

**Issue by SimonSchick** [sequelize/umzug#168]
Apparently it's not used and it's just bloating installations.

**Comment by SimonSchick** [comment]
Actually I just checked, it seems to be injected via `gulp-babel`.

I'm going to be bold and request that support for node

```
  - SEQUELIZE=^1.0.0
  - SEQUELIZE=^2.0.0
```

and NodeJS v4, v6 and v7 be dropped in favor of v8 and v9 and native async/await.

If done as a major release nobody will be affected by this.

**Comment by rtablada** [comment]
@SimonSchick while I would agree that v4 should be dropped, v6 is still supported and LTS.

While it limits native `async/await` in the actual `umzug` codebase, it doesn't stop users from using native async/await in their migrations if they are using v8+.

**Comment by SimonSchick** [comment]
Perhaps this could be done with the sequelize v5 release?

**Comment by SimonSchick** [comment]
I want to bring this back up, it seems rather questionable for umzug to be responsible for loading es6 modules.
ES6 modules are not even really supported in node (special handling, separation from commonjs etc.)
Shipping a large dependency such as `babel-runtime` just to serve a very small minority of people that for reasons unknown to me chose to use es6 modules for their migrations seems rather wasteful.
I think allowing to provide a loader interface, eg. a function that gets called and returns a promise with the up/down function would be much better and much less wasteful.

To summarize: The responsibility of module loading should lie with the consumer not the library itself, forcing everyone that uses umzug to install a bloated package like `babel-runtime` seems harmful.

**Comment by Floby** [comment]
As of April 2019, node 6 is not LTS anymore.
`babel-runtime` should be dropped from this package. Writing migrations in ES6 modules seems like a dubious usecase.

**Comment by FibreFoX** [comment]
@PascalPflaum as the linked PR #179 got closed, do you have any update to when babel-runtime can get removed?

It's a bit frustrating, because NodeJS LTS now is v12, long time ago since NodeJS 6

**Comment by tgabi333** [comment]
Here is my fork: 
https://github.com/TechTeamer/umzug

```
npm i @techteamer/umzug
```

ChangeLog:
```
removed CoffeScript support
removed babel-runtime dependency
removed node 6, 8 support, only node 10+ is supported
standardjs upgrade to v14
```

**Comment by sushantdhiman** [comment]
Fixed by https://github.com/sequelize/umzug/pull/202, released as `2.3.0`

**Comment by gvlasov** [comment]
Then how to we use migrations as ES6 modules now? For example, I want a migration like this:

```ts
import PostTypeId from "data/PostTypeId";

export async function up() {
  // Some code that usees PostTypeId
}

export async function down() {

}
```

Also please see my StackOverflow question: https://stackoverflow.com/questions/66301824/how-to-import-stuff-in-umzug-migrations

**Comment by gvlasov** [comment]
That turned out to be a silly quesiton, the real quesiton is how to use your own tsconfig.json for Umzug migrations, so they can use the same paths as the rest of my project.
